packages/paste-core/components/input-box/src/Prefix.tsx

```diff
@@ -19,13 +19,13 @@ export interface PrefixProps {
 
 const Prefix = React.forwardRef<HTMLDivElement, PrefixProps>(
   ({children, disabled, element = 'PREFIX', variant}, ref) => {
-    let backgroundColor = 'colorBackground' as BackgroundColor;
+    let backgroundColor = 'colorBackgroundWeak' as BackgroundColor;
     let borderColor = 'colorBorderWeaker' as BorderColor;
     if (disabled && variant === 'inverse') {
       backgroundColor = 'none';
       borderColor = 'colorBorderInverseWeaker';
     } else if (variant === 'inverse') {
```

Comment:
todo: add a conditional for a disabled state to apply these styles:
- no background color
- color-border-weaker